examples/asr/other/QuartzNetModel.ipynb

```diff
@@ -0,0 +1,454 @@
+{
```

Comment:
Why is this file in other? Can we also clear the outputs from the notebook?

Reply:
moved to just asr (maybe will merge it with another tutorial before release). also cleanup up outputs.